client/src/pages/organizations/Laydown.tsx

```diff
@@ -74,8 +74,6 @@ const OrganizationLaydown = ({
             exportTitle={
               readOnly ? null : `Organization diagram for ${organization}`
             }
-            width={width}
-            height={height}
```

Comment:
You now no longer respond to browser window resizes (or phone orientation swaps from portrait to landscape or vice versa); see the `useResizeDetector()` hook at the start of this component.